Added some marketing ish content here (#291)
casper-network/docs: source/docs/casper/dapp-dev-guide/why-build-on-casper.md

- [SDK Client Libraries](#sdk-client-libraries)
- [Low Gas Fees](#low-gas-fees)

## Thriving Ecosystem {#thriving-ecosystem}

> The Casper Ecosystem is growing every day, with new dApps and tools being added. Here is a short list of tools you can use.

Reviewer: ```suggestion
The Casper Ecosystem is growing every day through the addition of new dApps and tools. Here is a short list of tools you can use.
```

A small reconfiguring suggestion for sentence flow.

### Wallets
- [Ledger](https://www.ledger.com/supported-crypto-assets/token)
- [Tor.us](https://app.tor.us/)
- [Casper Signer](https://chrome.google.com/webstore/search/casper?hl=en)

### Block Explorers
- [cspr.live](https://cspr.live)
- [casperstats.io](https://casperstats.io)

### Developer Tools
- [casperholders.io](https://casperholders.io)

## Developer-Friendly Language {#developer-friendly-language}

> Casper Network's development ecosystem is designed to support WebAssembly, rather than being written in proprietary languages like Solidity. Casper contracts function just like regular software. This feature simplifies the development path for enterprises and development teams that want to build on the Casper Network.

Reviewer: ```suggestion
Casper Network's development ecosystem supports WebAssembly by design, rather than requiring proprietary languages like Solidity. Casper contracts function just like regular software. This feature simplifies the development path for enterprises and development teams that want to build on the Casper Network.
```

Another flow suggestion.

The Casper Network offers powerful accounts that are more than just public keys. Accounts offer weights for separate key management and transaction signing rights, and the ability to run session code (wasm) in the account context. By running session code in the account, it's possible to delegate transaction signing to multiple keys, revoke lost keys to recover accounts and store data within the account itself. It is also possible to securely share state between accounts and contracts (without expensive cryptographic checks). Refer to the [Casper Permissions Model](https://casper.network/docs/design/accounts#accounts-permissions) for more details.

## Contract Upgrades {#contract-upgrades}
Casper smart contracts use a package management model, which allows the direct upgrading of on-chain smart contracts, eliminating the need for complex migration processes and making it easy for developers to add new features or fix bugs by adding a new version of the contract. When installing a contract, it's possible to designate a contract as 'not upgradeable', which is suitable for DeFi contracts.

## Development Tools {#development-tools}

### IDE Integration {#ide-integration}

> The Casper development process is designed to be familiar to all developers. You can run and build code locally within an IDE and use assertions and tests to verify the functionality of your application. You can set the contract's starting state and create and run tests on your development machine. Casper contracts function like regular software, so there is very little you need to know about the blockchain to get started.

Reviewer: ```suggestion
The Casper development process strives to be familiar to all developers. You can run and build code locally within an IDE and use assertions and tests to verify the functionality of your application. You can set the contract's starting state and create and run tests on your development machine. Casper contracts function like regular software, so there is little you need to know about the blockchain to get started.
```

One last flow suggestion.
